streams/src/test/java/org/apache/kafka/streams/state/internals/StreamThreadStateStoreProviderTest.java

```diff
@@ -449,22 +454,18 @@ private StreamTask createStreamsTask(final StreamsConfig streamsConfig,
             new TopologyConfig(null, streamsConfig, new Properties()).getTaskConfig(),
             streamsMetrics,
             stateDirectory,
-            EasyMock.createNiceMock(ThreadCache.class),
+            mock(ThreadCache.class),
             new MockTime(),
             stateManager,
             recordCollector,
             context, logContext);
     }
 
     private void mockThread(final boolean initialized) {
-        EasyMock.expect(threadMock.isRunning()).andReturn(initialized);
-        EasyMock.expect(threadMock.allTasks()).andStubReturn(tasks);
-        EasyMock.expect(threadMock.activeTaskMap()).andStubReturn(tasks);
```

Comment:
I guess first three stubs are no longer needed or?

Reply:
Yes, apologies, I will add comments when I delete stubs like this in future pull requests. Mockito reported these as unused and when I commented them out in the original EasyMock tests the tests still passed.